scripts/hotrod-integration-test.sh

```diff
@@ -13,9 +13,10 @@ make prepare-docker-buildx
 #build image locally for integration test
 bash scripts/build-upload-a-docker-image.sh -l -c example-hotrod -d examples/hotrod -p "${platforms}"
 
-export CID=$(docker run -d -p 8080:8080 localhost:5000/$REPO:${GITHUB_SHA})
+export CID
+CID=$(docker run -d -p 8080:8080 localhost:5000/$REPO:"${GITHUB_SHA}")
 i=0
-while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' localhost:8080)" != "200" && ${i} -lt 30 ]]; do
+while [[ "$(curl -s -o /dev/null -w "%{http_code}" localhost:8080)" != "200" && ${i} -lt 30 ]]; do
```

Comment:
```
"$(curl -s -o /dev/null -w "
```
the last `"` breaks the string

Reply:
That was my mistake, you are right 
`
while [[ "$(curl -s -o /dev/null -w '%{http_code}' localhost:8080)" != "200" && ${i} -lt 30 ]]; do 
`
adding single quotes should work fine